code/compatibility/src/main/java/org/betonquest/betonquest/compatibility/holograms/decentholograms/DecentHologramsIntegrator.java

```diff
@@ -65,24 +65,30 @@ public BetonHologram createHologram(final Location location) {
     public void hook(final BetonQuestApi api) throws HookException {
         super.hook(api);
         if (!Bukkit.getPluginManager().isPluginEnabled("PlaceholderAPI")) {
-            log.warn("Holograms from DecentHolograms will not be able to use BetonQuest variables in text lines "
-                    + "without PlaceholderAPI plugin! Install it to use holograms with variables!");
+            log.warn("Holograms from DecentHolograms will not be able to use BetonQuest placeholders in text lines "
+                    + "without PlaceholderAPI plugin! Install it to use holograms with placeholders!");
         }
     }
 
+    /**
+     * Parses a BetonQuest placeholder with package and converts it to the appropriate format for DecentHolograms
+     * which uses PlaceholderAPI format.
```

Comment:
```suggestion
     * Parses a package-specific BetonQuest placeholder and converts it to the PlaceholderAPI format
     * since DecentHolograms requires it.
```